package.json

```diff
@@ -15,6 +15,7 @@
     "gen": "ts-node src/scripts/generate.ts",
     "prepare": "npm run build",
     "antlr4ts": "antlr4ts -visitor src/antlr/SqlBase.g4",
+    "postantlr4ts": "eslint src/antlr/**/*.ts --fix && prettier src/antlr/**/*.ts --write",
```

Comment:
@pavelgubin this now runs linting and formatting after every code generation. It finds no warnings / errors in the generated files if I run it. If a new warning / error appears, this could be a sign for us to investigate.